### ISSUE TITLE
Refactor pytest fixtures into a pytest-growler plugin

### ISSUE THREAD

**Issue by lmergner** [pyGrowler/Growler#13]
I'm copying tests from growler-mako into my extension, so it's probably a good idea to eventually consolidate common fixtures into a pytest plugin at some future point.

**Comment by akubera** [comment]
Yes, this is on my (long) to-do list for Growler.

**Comment by lmergner** [comment]
I figured. Is it helpful to leave these ideas as issues?

**Comment by akubera** [comment]
Yes I believe so. Besides offering a place for discussion, a public list of things that need to be worked on is a good motivator.

What I really need for this issue is a person or guide that can explain best practices for organizing a good pytest module. It looks like I was starting to do a refactor in [tests/mock_classes.py](https://github.com/pyGrowler/Growler/blob/dev/tests/mock_classes.py), but that was about a year ago. As growler has changed a lot since I started it (>2 years!?! my, how time flies) I haven't really worried about adding support libraries. Since it's starting to solidify and I'm pretty happy with its structure, I will probably look into this (assuming people use it, making it worth my time).
